### ISSUE TITLE
tooth number 12 is not responding

### ISSUE THREAD

**Issue by jaipasi** [cgmcintyr/dc-demo#1]
tooth-12 is not clickable,
it doesn't respond to event listener
and it will be great if make another Pediatric dental chart
thank you ....

**Comment by cgmcintyr** [comment]
Hi, this was for a job interview, and the code is a complete hack. If you want to add features I can review any pull requests, but I don't plan on working on this myself :)

**Comment by jaipasi** [comment]
Yeah i did some modifications ... but still im a learner ... So cant go that far adding features. ... pulling request may help ... thanks for the response.☺
